test/runtests.jl

```diff
@@ -25,4 +25,30 @@ using Test
         @test nwscore(nothing, base; gap = -2) == -2
     end
 
+    
+    @test nwscore('A', 'T') == -1
+    @test nwscore('A', 'T'; mismatch=-2, match = 3) == -2
+    @test nwscore('A', 'T'; mismatch=-2, match = 3, gap=-2) == -2
+    @test nwscore(nothing, 'C'; mismatch=-1, match=1, gap =-2) == -2
+    @test nwscore(nothing, 'C'; mismatch=-1, match=1) == -1
+    @test nwscore('A', nothing; mismatch=-1, match=1) == -1
+    @test nwscore(nothing, 'C'; gap =-2) == -2
+
+    @test_throws ArgumentError nwscore(nothing, nothing)
+    @test_throws ArgumentError nwscore(nothing, nothing; gap=-2)
+    @test_throws ArgumentError nwscore(nothing, nothing; mismatch=-1, match=1, gap =-2)
+
+
+    
+end
+
+@testset "Lab03" begin
+    #not sure if this is what is meant by scoring system
+    @test nwalign("AGGT", "ACGAT") == "ACGAT/AGG-T" #returns alignment, but is that a string? best way to show both sequences?
```

Comment:
Yeah - there are lots of possibilities. Consider also that you might have multiple "best" alignments. My advice would be to have a structure that's stable - here you have two strings separated by `/`, but in the next example you just have one, why not `"AGT/AGT"`?

You might also consider a Tuple (`("ACGAT", "AGG-T")`) or an array of Tuples in case you have multiple best alignments. Also consider - will you return the score of the alignment?